backend/app/controllers/admin-login.controller.js

```diff
@@ -0,0 +1,54 @@
+const config = require("../config/auth.config");
+const db = require("../models");
+const User = db.user;
+const Role = require("../models/role.model");
+
+var jwt = require("jsonwebtoken");
+var bcrypt = require("bcryptjs");
+const { isAdmin } = require("../middlewares/authJwt");
+
+exports.adminLogin = (req, res) => {
+  User.findOne({
+    username: req.body.username
+  })
+    .populate("roles", "-__v")
+    .exec((err, isAdmin) => {
```

Comment:
I don't really understand how `isAdmin` works here. Is that really the second argument here?